### DISCUSSION TITLE
Nova doesn't looks nice

### DISCUSSION THREAD

**Discussion by Zen0x7** [laravel/nova-issues#6542] [Q&A]
- Laravel Version: #.#.#
- Nova Version: #.#.#
- PHP Version: #.#.#
- Database Driver & Version:
- Operating System and Version: #.#
- Browser type and version: #.#
- Reproduction Repository: https://github.com/###/###

### Description:

This isn't a huge problem but it's a point that can be take in care.

![image](https://github.com/user-attachments/assets/56f3eec0-ba00-428c-add0-915e4ac68bec)

That's how looks like in a wide-widescreen. There are a way to solve this?

**Comment by Andrewrafat** [comment]
good

**Reply by Andrewrafat** [reply]
💗

**Comment by lucasff** [comment]
I don't see the point here? It does look nice, it looks exactly like it should, stretched.

**Reply by Zen0x7** [reply]
Yeah, maybe the point isn't clear at all, it's something that can be inferred.

By the way, from my point of view, it's quite hard to me get focused on stretched applications, because the center point of my eyes can't focus, increasing the effort to engage the content, consider this as a accessibility issue, going further, it's produces me vertigo.
